package-lock.json

```diff
@@ -340,7 +340,8 @@
             "code-point-at": {
               "version": "1.1.0",
               "bundled": true,
-              "dev": true
+              "dev": true,
```

Comment:
Since you haven't added any new packages to the project, I'd request that you revert all the changes to this file.